### PR TITLE
feat(general): #10 Added material design.

### PR DESCRIPTION
**Pull request by VenkatVenkatesh**
1. Added material design to the app using `ng add` from the CLI
2. Created a generic module to import material design.
3. Imported the generic module into `app.module.ts`.

### CODE REVIEW

File: web/angular.json [hunk 1]
```diff
@@ -137,4 +139,4 @@
     }
   },
   "defaultProject": "TribeDashboard"
-}
+}
```
Comment (eddiejaoude) [inline web/angular.json]: Ending blank is required

File: web/src/app/app.module.ts [hunk 2]
```diff
@@ -3,14 +3,17 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { AppMaterialModule } from './app.material-module';
+
```
Comment (eddiejaoude) [inline web/src/app/app.module.ts:6]: I don't think 2 blank lines are needed here

Comment (VenkatVenkatesh) [reply web/src/app/app.module.ts:6]: okay Eddie. Will remove it.

Comment (eddiejaoude) [reply web/src/app/app.module.ts:6]: Thank you 👍

File: web/angular.json (hunk 1 above)
Comment (VenkatVenkatesh) [reply web/angular.json]: Resolved @eddiejaoude

**Comment by eddiejaoude** [comment]
Looks good. Can you use and add a screenshot of using 1 Material Design component to show that it all works 🤓

**Comment by VenkatVenkatesh** [comment]
Sure @eddiejaoude . Will add and let you know. :+1:

**Comment by VenkatVenkatesh** [comment]
localhost_4200_ (1)

**Comment by VenkatVenkatesh** [comment]
Hi @eddiejaoude , I have attached the screen shot and worked on the review comments as well.

**Comment by eddiejaoude** [comment]
Hi @VenkatVenkatesh , some comments are not addressed yet, are you still working on it?

Can you check spacing and indentation. 

Also check you have the EditorConfig plugin installed as it will fix everything for you.

**Comment by VenkatVenkatesh** [comment]
Hi @eddiejaoude , I have installed editorConfig and resolved all the issue. Could you please check it out ?

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Comment by eddiejaoude** [comment]
Can you please move the ticket across on the project board and close it https://github.com/DashboardHub/TribeDashboard/projects/1#card-19860092

**Comment by VenkatVenkatesh** [comment]
Okay @eddiejaoude .

**Pull request merged**